main.mk

```diff
@@ -1,4 +1,5 @@
 INSTANCE_ID = main 
 VARS_FILE = vars_main.yaml
+PRINT_OUTPUT = /var/lib/tomcat/webapps
 
```

Comment:
the make file now copies the war to the correct directory, but the created file is 700 and tomcat won't unpack it unless i set it to 755 manually...